include/game/bases/d_a_player_manager.hpp

```diff
@@ -3,14 +3,21 @@
 
 class daPyMng_c {
 public:
+    /// @unofficial
+    enum PyType {
+        TYPE_MARIO, TYPE_LUIGI, TYPE_YELLOW_TOAD, TYPE_BLUE_TOAD
```

Comment:
Use the PLAYER_CHARACTER_e enum values if possible (see previous comment).

Reply:
Note that the yellow and blue toad values are swapped here, so I think it has to be something else. It's used as an argument to `80060110` / `int daPyMng_c::getPlayerIndex(daPyMng_c::PyType)` unofficially